package.json

```diff
@@ -824,11 +824,13 @@
     "azure-iot-device-mqtt": "^1.15.0",
     "azure-iothub": "^1.12.4",
     "body-parser": "^1.18.2",
+    "dummy-json": "^3.0.4",
     "express": "^4.16.3",
     "fecha": "^4.2.0",
     "fs-extra": "^9.0.1",
     "glob": "^7.1.6",
     "handlebars": "^4.7.7",
+    "keytar": "^7.9.0",
```

Comment:
File is referenced in code (but must have been a nested dependency of another package, this way we can ensure major package versions and fix dependency vulnerabilities.